src/main/java/com/ksm/wstbackoffice/controller/CountryController.java

```diff
@@ -44,4 +47,20 @@ public CountryController(CountryService countryService) {
         return ResponseEntity.status(HttpStatus.OK)
                 .body(country);
     }
+
+    @PostMapping("/start-with")
+    public ResponseEntity<Map<String, HashMap<String, ?>>> findCountriesStartWith(@RequestBody List<String> stringsToStartWith) {
+        if (stringsToStartWith.isEmpty()) {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
+        }
+
+        Map<String, HashMap<String, ?>> filteredCountries = countryService.startWith(stringsToStartWith);
```

Comment:
`
Map<String, HashMap<String, ?>>
`
Why do you specify HashMap. Is it possible to use Map twice?
HashMap is an implementation of the Map interface. It is always better to specify an interface as a variable type and specify implementation as the value you assign to it.
E.g.
Instead of this
`
HashMap someMap = new HashMap();
`use this
`
Map someMap = new HashMap();
`

Please fix this in all your PR

Reply:
done